user/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import DoubleCheckView,SignUpView,SignInView
+urlpatterns=[
+    path('/doublecheck',DoubleCheckView.as_view()),
+    path('/signupview',SignUpView.as_view()),
```

Comment:
endpoint에 'view' 용어가 포함되면, 불필요하게 url이 길어질 수 있습니다.
```suggestion
    path('/signup',SignUpView.as_view()),
```

Reply:
수정완료